template/src/components/Error/NotFoundPage.test.js

```diff
@@ -0,0 +1,61 @@
+import { getBackToHomePath } from './NotFoundPage';
```

Comment:
If we trash the go back to home stuff above, this can also go away. Though, thanks for adding the tests.